mmv1/third_party/terraform/services/filestore/resource_filestore_instance_test.go.tmpl

```diff
@@ -473,9 +474,35 @@ func TestAccFilestoreInstance_replication(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: testAccFilestoreInstance_replication(context),
+				Check: resource.ComposeTestCheckFunc(
+/*
+at this time the effective_replication is not available in the API, I think reason being it first creates the source_instance and then fetches the result
+then it creates the replica_instance and fetches the result for the replica_instance, but doesn't re-fetch and update state of the source_instance that now has an effective_replication
+					resource.TestCheckResourceAttr(
+						"google_filestore_instance.source_instance",
+						"effective_replication.0.replicas.0.peer_instance",
+						"projects/" + context["project"].(string) + "/locations/us-east1/instances/tf-test-replica-instance" + context["random_suffix"].(string),
+					),
+					resource.TestCheckResourceAttr(
+						"google_filestore_instance.source_instance",
+						"effective_replication.0.role",
+						"ACTIVE",
+					),
+*/
```

Comment:
yeah, I think you're exactly right in terms of why the test is failing. Checks run after the update, but there was no update to the source, so it won't have the new state until the next refresh. Let's just leave this off for now.

```suggestion
```